docs/blog/newsletter-june-2019.rst

```diff
@@ -0,0 +1,142 @@
+
+-------------------------------------------------------
+Order from chaos, or, a conversation about docs cleanup
+-------------------------------------------------------
+
+One generous WtD-er shared a blog post they'd written about organizing and curating doc sets, and the post generated a chorus of thanks and expressions of shared struggle. One notable piece of advice was to seek out and make public the often-hidden or kept-private stashes of information that individuals in a company invariably gather. Here's the blog post: https://medium.com/@brooke.wayne/how-to-organize-your-docs-bd797c616b48
+
+Over the years, this has been a really popular topic at Write the Docs conferences. If it's a problem you're interested in, there's an embarassment of riches for you in the video archives:
+
+- `What can brownfield do for you? </videos/na/2015/what-can-brownfield-do-for-you-mnishiyama>`__ - Mo Nishiyama, Portland 2015
+- `User-story driven docs </videos/na/2015/user-story-driven-docs-jfernandes>`__ - Joao Fernandes, Portland 2015
+- `MacGyvering your docs </videos/eu/2015/macgyvering-your-docs-proeland>`__ - Paul Roeland, Prague 2015
+- `Bootstrapping docs at a startup </videos/na/2017/bootstrapping-docs-at-a-startup-jesse-seldess>`__ - 
+Jesse Seldess, Portland 2017
+- `Backseat content strategy <https://www.youtube.com/watch?v=omwamA30e_Y&list=PLy70RNJ7dYrJ1wANiqa7ObwUnoJjouQjt&index=4>`__ - Laura Bailey, Australia 2018
+- `Where do I start? The art and practice of documentation triage </videos/portland/2018/where-do-i-start-the-art-and-practice-of-documentation-triage-neal-kaplan>`__ - Neil Kaplan, Portland 2018
+- `How to tear down existing documentation and rewrite docs that actually work </videos/prague/2018/how-to-tear-down-existing-documentation-and-rewrite-docs-that-actually-work-alexandra-white>`__ - Alexandra White, Prague 2018
+- `Tackling technical debt in the docs </videos/prague/2018/tackling-technical-debt-in-the-docs-louise-fahey>`__ - Louise Fahey, Prague 2018 
+- `A year in the life of the better docs project </videos/prague/2018/a-year-in-the-life-of-the-better-docs-project-rowan-cota>`__ - Rowan Cota, Prague 2018
+
+--------------------
+DITA vs docs-as-code
+--------------------
+
+Our final discussion this month comes from a community question about what the differences are between docs-as-code and DITA. The short answer is that docs-as-code is a practice, whereas DITA is a tool and a format. So in theory, you could practise docs-as-code with DITA just as you would with markdown.
+
+However, it's not necessarily that straightforward. One issue is that if you chose to use DITA, you put a higher barrier to entry on non-tech-writer contributions: reading the XML isn't tricky, and licenses for tools can be expensive. You can help others review by publishing the content to PDF, but that may not be an ideal review format.
```

Comment:
"reading the XML is tricky" ?? (currently "isn't")

Reply:
Oops!